### PR TITLE
fix some typos for plugin

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

**- What I did**
fix some typos for plugin

### CODE REVIEW

File: docs/reference/commandline/plugin_create.md [hunk 1]
```diff
@@ -36,16 +36,14 @@ The following example shows how to create a sample `plugin`.
 
 ```bash
 $ ls -ls /home/pluginDir
-
+total 4
 4 -rw-r--r--  1 root root 431 Nov  7 01:40 config.json
 0 drwxr-xr-x 19 root root 420 Nov  7 01:40 rootfs
 
 $ docker plugin create plugin /home/pluginDir
-
```
Comment (AkihiroSuda) [inline docs/reference/commandline/plugin_create.md]: Empty lines are added consistently across all the files in the `commandline` directory

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/plugin_create.md (hunk 1 above)
Comment (yuexiao-wang) [inline docs/reference/commandline/plugin_create.md]: It is inconsistent with docker commands output.  Why added empty lines?

Comment (AkihiroSuda) [inline docs/reference/commandline/plugin_create.md]: so that input can be distinguished from the output, I guess

Comment (yuexiao-wang) [inline docs/reference/commandline/plugin_create.md]: Well.  I will recover these empty lines

**Comment by yuexiao-wang** [comment]
@AkihiroSuda  
It is updated and PTAL

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**